lib/sisimai/bite/email/activehunter.rb

```diff
@@ -0,0 +1,124 @@
+module Sisimai::Bite::Email
+  module Activehunter
+    # Sisimai::Bite::Email::Activehunter parses a bounce email which created
+    # by TransWARE Active!hunter.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Activehunter.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\A"MAILER-DAEMON"/,
+        :subject => %r/FAILURE NOTICE :/,
+      }
+      Re1 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->